### DISCUSSION TITLE
4.2.7 Asset Missing

### DISCUSSION THREAD

**Discussion by bluedevil678** [mriscoc/Ender3V2S1#1167] [Q&A] [answered]
The asset list only shows: 

[Ender3V2-422-BLTUBL-MPC-20231202.bin]
[Ender3V2-422-MM-MPC-20231202.bin]
Source code (zip)
Source code(tar.gz)

However the Ender3V2-427-BLTUBL-MPC seems to be missing - is this a known issue ?

**Comment by Homer-S** [comment]
They appear to be "DEMO" only at the moment.

![image](https://github.com/mriscoc/Ender3V2S1/assets/1320262/7b24b4ad-94a0-4cc0-956d-9a936752f3ef)

https://github.com/mriscoc/Special_Configurations/releases

Homer

**Reply by bluedevil678** [reply]
Well spotted! Demo states are limited to 1h of print time. I may just be impatient so I'll check back often.

**Comment by mriscoc** [comment, marked as answer]
Versions 4.2.7 have been released.

**Reply by bluedevil678** [reply]
A massive thank you!
